sky/backends/backend_utils.py

```diff
@@ -482,12 +482,34 @@ def add_cluster(
         sky_autogen_comment = ('# Added by sky (use `sky stop/down '
                                f'{cluster_name}` to remove)')
         overwrite = False
-        overwrite_begin_idx = None
         ip = ips[0]
         if docker_user is not None:
             ip = 'localhost'
 
         config_path = os.path.expanduser(cls.ssh_conf_path)
+        with open(config_path) as f:
+            config = f.readlines()
+
+        ssh_dir = cls.ssh_cluster_path.format('')
+        if not os.path.exists(os.path.expanduser(ssh_dir)):
+            os.makedirs(os.path.expanduser(ssh_dir))
+            # Handle Include on top of Config file
+            include_str = f'Include {cls.ssh_cluster_path.format("*")}'
```

Comment:
Why do we only check and add it when `ssh_dir` not exist?

Reply:
Good catch! I think we should add it regardless for backward compatibility. Fixed now!